app/src/main/java/travelbook/android/app/data/MemoriesDao.java

```diff
@@ -1,5 +1,10 @@
 package travelbook.android.app.data;
 
+import android.app.Activity;
```

Comment:
Unused import.
Keep your code clean: Android Studio highlights warnings in a file and you must respect to them.